### PR TITLE
Adjust and complete the instructions in the "Getting started" chapter

### PR DESCRIPTION
**Pull request by DelazJ**
making the experience smoother
Use atomic steps for instructions
Add more details here and there

### Description

Goal: improve getting started experience at https://docs.qgis.org/testing/en/docs/user_manual/introduction/getting_started.html

Ticket: #

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/user_manual/introduction/getting_started.rst [hunk 1]
```diff
@@ -40,9 +46,10 @@ instructions. They are distributed with the QGIS source code in a file
 called :file:`INSTALL`. You can also find them online at
 https://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html.
 
-If you want to build a particular release, you should replace ``master`` by the
-release branch (commonly in the ``release-X_Y`` form) in the above-mentioned
-link because instructions may differ.
+
+If you want to build a particular release and not the version in development,
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
If you want to build a particular release,
```

File: source/docs/user_manual/introduction/getting_started.rst [hunk 2]
```diff
@@ -40,9 +46,10 @@ instructions. They are distributed with the QGIS source code in a file
...
+you should replace ``master`` with the release branch (commonly in the
+``release-X_Y`` form) in the above-mentioned link because instructions may differ.
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
``release-X_Y`` form) in the above-mentioned link (installation instructions may differ).
```

File: source/docs/user_manual/introduction/getting_started.rst [hunk 3]
```diff
@@ -53,14 +60,18 @@ that overrides the default :ref:`user profile <user_profiles>` path and forces
...
 
 .. index:: Data sample
 .. _label_sampledata:
 
 Downloading sample data
 -----------------------
 
-This user guide contains examples based on the QGIS sample dataset.
+This user guide contains examples based on the QGIS sample dataset (also called
+``Alaska dataset``).
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
the ``Alaska dataset``).
```

File: source/docs/user_manual/introduction/getting_started.rst [hunk 4]
```diff
@@ -142,48 +152,74 @@ To stop QGIS, use:
...
+   #. Click on the |symbology| :guilabel:`Symbology` tab
+   #. Select blue as fill color.
+   #. Press :guilabel:`Apply`. Lakes are now displayed in blue in the map canvas.
+#. To display the name of the lakes:
+
+   #. Click on the |labeling| :guilabel:`Labels` tab
+   #. Select :guilabel:`Single labels` in the drop-down menu to enable labeling.
+   #. From the :guilabel:`Label with` list, choose the ``NAMES`` field.
+   #. Press :guilabel:`Apply`. Names will now load over the boundaries.
+#. You can improve readability of the labels, adding a white buffer around them:
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
#. You can improve readability of the labels by adding a white buffer around them:
```

**Comment by lajunek** [comment]
@DelazJ Do we have Conflicting files because I also worked on this chapter? -> #3951

**Pull request merged**